modules/watsonx-data-v2/pom.xml

```diff
@@ -13,15 +13,18 @@
     <name>Watsonxdata</name>
     <packaging>jar</packaging>
 
+    <properties>
+        <maven.javadoc.skip>true</maven.javadoc.skip>
+    </properties>
+
     <dependencies>
+        <!-- There should be a dependency for each module whose request example class exists in this "examples" module. Add 
+            new "dependency" entries here as needed when you add a request examples class for a new service. Note: the "artifactId" values 
+            below should reflect the "artifactId" value specified in each module's pom.xml file (i.e. not the module's directory name). -->
         <dependency>
             <groupId>com.ibm.cloud</groupId>
-            <artifactId>sdk-core</artifactId>
```

Comment:
Why need to change this file?

Reply:
@fanfeigithub i reverted that change. Could you please review it again?